docs/chapter_4/lesson_11.md

```diff
@@ -0,0 +1,56 @@
+Goal:
+* Learners should understand more information(params) in genesis.json
+
+# Lesson 11: Introduction to genesis.json
+
+genesis.json: Is a json file that defines the initial state of the chain, as block 0 of the chain. In this file, it will contain a lot of necessary information such as initialization time, important parameters like crisis, consensus param, bank, ...
```

Comment:
genesis.json: Is a json file that defines the initial state of the chain, as block 0 of the chain. This file will contain necessary information related to network identification (chain_id), consensus params, and parameter configuration for keepers.